toolbox/cluster/set_scale.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [ -z "${2-}" ] && [ -z "${3-}" ]; then
```

Comment:
I don't think this is the intended tests: if the 2nd and 3rd arguments are empty --> usage

maybe: `[[ -z "${1:-}" || -z "${2:-} || "${3:---force}" != "--force""`?
if 1 or 2 are empty, or 3 isn't set or is equal to `--force`